src/Main.elm

```diff
@@ -143,11 +135,15 @@ saveAction : Model -> Action -> Model
 saveAction model action =
     case model.actionA of
         Nothing ->
-            { model
-                | valueA = displatToFloat model.display
-                , actionA = Just action
-                , displayStatus = ShowingResults
-            }
+            if action == Equals then
+                model
```

Comment:
Is it just me, or does this make the `Equals` button do nothing all the time?

Reply:
Yes, it does nothing at the moment.